_data/backup.yml

```diff
@@ -124,9 +124,10 @@ websites:
 
   - name: Mega
     url: https://mega.nz/
-    twitter: MEGAprivacy
     img: mega.png
-    tfa: No
+    tfa: Yes
+    software: Yes
+    doc: https://mega.nz/help/client/webclient/security-and-privacy/5bb5436cf1b70966038b456b 
```

Comment:
There's a blank space at the end of the doc line. Please remove that

Reply:
I've removed the blank space from end of the doc line.